backtrace-library/src/main/cpp/backends/crashpad-backend.cpp

```diff
@@ -219,10 +236,28 @@ void ReEnableCrashpad() {
     // Re-enable uploads if disabled
     if (disabled) {
         if (database == nullptr) {
-            __android_log_print(ANDROID_LOG_ERROR, "Backtrace-Android", "Crashpad database is null, this should not happen");
+            __android_log_print(ANDROID_LOG_ERROR, "Backtrace-Android",
+                                "Crashpad database is null, this should not happen");
             return;
         }
         database->GetSettings()->SetUploadsEnabled(true);
         disabled = false;
     }
 }
+
+bool EnableCrashLoopDetectionCrashpad() {
+    __android_log_print(ANDROID_LOG_ERROR, "Backtrace-Android", "Inside EnableCrashLoopDetectionCrashpad");
+    if (client != nullptr) {
+        return client->EnableCrashLoopDetection();
+    } else {
```

Comment:
does it make sense to print warning the client is not available? We need to be clear in client logs to show what doesn't work. We need to show it fast and hard